pkg/k8shandler/collection.go

```diff
@@ -505,7 +505,8 @@ func updateFluentdDaemonsetIfRequired(desired *apps.DaemonSet) (err error) {
 		if flushBuffer {
 			current.Spec.Template.Spec.Containers[0].Env = append(current.Spec.Template.Spec.Containers[0].Env, v1.EnvVar{Name: "FLUSH_AT_SHUTDOWN", Value: "True"})
 			if err = sdk.Update(current); err != nil {
-				return fmt.Errorf("Failed to prepare Fluentd daemonset to flush its buffers: %v", err)
+				logrus.Errorf("Failed to prepare Fluentd daemonset to flush its buffers: %v", err)
```

Comment:
I don't think you need to print the error here - instead, print the error as the error return from the `RetryOnConflict` - if you really need to print something here, make it `Debugf` instead.